examples/usecases/ranking_with_multitask_learning.ipynb

```diff
@@ -0,0 +1,1663 @@
+{
```

Comment:
<p>Why do we combine CGC and PLE? I would expect another header PLE as it has its own API./ description and parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/948/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>The reason is that both CGC and PLE were introduced in the same paper. PLE is the multi-level version of CGC.</p><p>I have added this info to the beginning of that section:  "The&nbsp;<strong>CGC</strong>&nbsp;and&nbsp;<strong>PLE</strong>&nbsp;architectures were introduced in this&nbsp;<a href="https://dl.acm.org/doi/10.1145/3383313.3412236" target="_blank">p</a>aper&nbsp;(2020)."</p><p>Also added subsections for CGC and PLE</p><p>Does that help to clarify?</p>

Reply:
<p>Yes, I like it with the subheading. I understand that you wrote subheading per paper. I think more from the user perspective - as an user, I am interested in the different architecture designs and then test them (all or 1-by-1) and might prioritize them - independent if they are from the same paper or different papers?</p>